go/vt/vttablet/queryservice/fakes/stream_health_query_service.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 
+	"github.com/youtube/vitess/go/sqltypes"
```

Comment:
github.com/youtube -> vitess.io. You need to rebase and change all new imports :).

Reply:
yeah :-)

Reply:
done!